### PR TITLE
adding android cta

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Closes https://github.com/mozilla/webmaker-core/issues/295

File: src/pages/player/player.jsx [hunk 1]
```diff
@@ -85,6 +87,11 @@ module.exports = React.createClass({
         }
       }
     }
+
+    // Hide Android app ad when "No thanks" is clicked
+    React.findDOMNode(this.refs.dismissButton).addEventListener('click', function (event) {
```
Comment (k88hudson) [inline src/pages/player/player.jsx]: why can't you do this with an onClick?

File: src/pages/player/player.less [hunk 2]
```diff
@@ -43,4 +43,50 @@
       line-height: 28px;
     }
   }
+
+  .micro-modal {
+    display: none;
+
+    @media (min-width: 960px) {
```
Comment (k88hudson) [inline src/pages/player/player.less:50]: Why do we only show this on desktops?

**Comment by k88hudson** [comment]
Just a couple of questions/clarifications, looking good

File: src/pages/player/player.less (hunk 2 above)
Comment (gvn) [inline src/pages/player/player.less:50]: That was a decision from design. I guess it takes up too much space in a narrow viewport?

Comment (k88hudson) [inline src/pages/player/player.less:50]: cool, thanks

**Pull request merged**